elasticsearch/includes/classes/Elasticsearch.php

```diff
@@ -21,10 +20,10 @@ protected function load_dependencies() {
 		 * Load ES Health command class
 		 */
 		if ( defined( 'WP_CLI' ) && WP_CLI ) {
-			require_once __DIR__ . '/commands/class-health-command.php';
+			require_once __DIR__ . '/commands/HealthCommand.php';
```

Comment:
In general I do prefer this type of naming scheme, but the "WordPress way" is to prefix class files with `class-`, so we should follow that convention as end users will find it most familiar.